cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help

```diff
@@ -1,21 +1,21 @@
 NAME
-        Ballerina pull - Download modules from Ballerina Central
+        ballerina pull - Pull module from Ballerina Central
 
 SYNOPSIS
         ballerina pull <org-name>/<module-name>[:version]
 
 DESCRIPTION
-         Downloads the given module from Ballerina Central
-         (https://central.ballerina.io/) repository and installs to home
-         repository. By default, the module will be placed within the
+         Pull the given module from Ballerina Central
+         (https://central.ballerina.io/) repository and install to home
+         repository. By default, the module will be placed in the
          .ballerina home repository of the user.
 
 OPTIONS
          Ballerina pull command takes no options.
 
 EXAMPLES
-         Download Gmail connector by WSO2 from Ballerina Central
+         Pull Gmail connector by WSO2 from Ballerina Central
          $ ballerina pull wso2/gmail
 
-         Download Bar module by Foo organization from Ballerina Central
+         Pull Bar module by Foo organization from Ballerina Central
```

Comment:
```suggestion
         Pull the bar module by foo organization from Ballerina Central
```

Reply:
I wonder if adding Articles is the right thing here.